package.json

```diff
@@ -62,7 +63,7 @@
     "typescript": "^3.8.3"
   },
   "dependencies": {
-    "@cmsgov/design-system": "2.5.0",
+    "@cmsgov/design-system": "file:../design-system/packages/design-system",
```

Comment:
The PropTypes support for legacy applications not using typescript works great. 
I did revert this back to `"@cmsgov/design-system": "2.5.0",` for the testing.